apps/api/src/app/subscribers/e2e/update-subscriber.e2e.ts

```diff
@@ -54,4 +54,46 @@ describe('Update Subscriber - /subscribers/:subscriberId (PUT)', function () {
     expect(createdSubscriber.email).to.equal('changed@mail.com');
     expect(createdSubscriber.phone).to.equal('+972523333333');
   });
+
+  it('should update an existing subscriber credentials', async function () {
+    await axiosInstance.post(
+      `${session.serverUrl}/v1/subscribers`,
+      {
+        subscriberId: '123',
+        firstName: 'John',
+        lastName: 'Doe',
+        email: 'john@doe.com',
+      },
+      {
+        headers: {
+          authorization: `ApiKey ${session.apiKey}`,
+        },
+      }
+    );
+
+    const response = await axiosInstance.put(
+      `${session.serverUrl}/v1/subscribers/123`,
```

Comment:
```suggestion
      `${session.serverUrl}/v1/subscribers/123/credentials`,
```

Reply:
Resolved in 0bcabdd5b03c5d7edac3671ab2d1bd1ffa7ca814